lib/buffer.js

```diff
@@ -131,33 +128,23 @@ const bufferWarning = 'The Buffer() and new Buffer() constructors are not ' +
                       'Buffer.allocUnsafe(), or Buffer.from() construction ' +
                       'methods instead.';
```

Comment:
This needs to be more explicit on how to fix this problem. This specific text does not guide the user on resolving the actual problem itself, we might want to add a link to a guide on how to update the relevant code.